LambdaEngine/Source/Game/ECS/Systems/Rendering/RenderSystem.cpp

```diff
@@ -1134,6 +1131,27 @@ namespace LambdaEngine
 			m_TLASDirty = true;
 		}
 
+		// Remove extension
+		{
+			// Fetch the current instance and its extension index.
```

Comment:
Unnecessary comment, the code speaks for itself here

Reply:
Not true, I think this is better. Can see more clearly what the whole code block does.